azure-communication-ui/build.gradle

```diff
@@ -51,11 +51,16 @@ buildscript {
             return rootProject.hasProperty("disableTaskRootCheck") && rootProject.getProperty("disableTaskRootCheck")
         }
 
-        compose_version = '1.3.1'
+        androidx_lifecycle_runtime_ktx_version = '2.3.1'
+        androidx_activity_compose_version = '1.5.1'
```

Comment:
@Mohtasim I think these are included for calling too? are we creating separate version for Chat? I think we will use same version.

Reply:
This is newly added. Calling uses a different one. $androidx_lifecycle_viewmodel_ktx_version